espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Http/Server.java

```diff
@@ -27,7 +30,8 @@ public Server() throws IOException, ServerErrorException {
     @Override
     public Response serve(IHTTPSession session) {
         BaseResponse response = router.route(session);
-        Gson gson = new Gson();
-        return newFixedLengthResponse(response.getHttpStatus(), "application/json", gson.toJson(response.getResponse()));
+        GsonBuilder gsonBuilder = new GsonBuilder();
+        gsonBuilder.registerTypeAdapter(AppiumStatus.class, new AppiumStatusAdapter());
+        return newFixedLengthResponse(response.getHttpStatus(), "application/json", gsonBuilder.create().toJson(response.getResponse()));
```

Comment:
can we use http://docs.oracle.com/javaee/6/api/javax/ws/rs/core/MediaType.html to represent media type constants?